Tests/Marketplace/marketplace_services.py

```diff
@@ -2365,6 +2371,44 @@ def is_unified_integration(self, file_path: str):
             os.path.basename(file_path).endswith('.yml')
         ])
 
+    def update_changelog_with_bc(self, changelog: Dict[str, Any]) -> None:
+        """
+        Receives changelog, checks if there exists a BC version in each changelog entry (as changelog entry might be
+        zipped into few RN versions, check if at least one of the versions is BC). If such version exists, adds a
+        true value to 'breakingChanges' field.
+        This function iterates every entry in changelog because it takes into consideration four scenarios:
+          a) Entry without breaking changes, changes to entry with breaking changes (because at least one of the
+             versions in the entry was marked as breaking changes).
+          b) Entry without breaking changes, does not change.
+          c) Entry with breaking changes, changes to entry without breaking changes (because all the BC versions
+             corresponding to the changelog entry were re-marked as not BC).
+          d) Entry with breaking changes, does not change.
+        Args:
+            changelog (Dict[str, Any]): Changelog data represented as a dict.
+
+        Returns:
+            (None): Modifies changelog, adds bool value to 'breakingChanges' field to every changelog entry, according
+                    to the logic described above.
+        """
+        predecessor_version: LooseVersion = LooseVersion('0.0.0')
+        for rn_version in sorted(changelog.keys(), key=LooseVersion):
+            rn_loose_version: LooseVersion = LooseVersion(rn_version)
+            changelog[rn_version]['breakingChanges'] = self.changelog_entry_contains_bc_version(predecessor_version,
+                                                                                                rn_loose_version)
```

Comment:
```suggestion
            changelog[rn_version]['breakingChanges'] = self.changelog_entry_contains_bc_version(predecessor_version,
                                                                                                rn_loose_version)
```
why do we update every entry? let's just update the relevant entries